Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -648,15 +648,18 @@ public double GetNamedSize(NamedSize size, Type targetElementType, bool useOldSi
 
 			public async Task<Stream> GetStreamAsync(Uri uri, CancellationToken cancellationToken)
 			{
-				using (var client = new HttpClient())
-				using (HttpResponseMessage response = await client.GetAsync(uri, cancellationToken))
+				using (var client = new HttpClient())				
 				{
+					HttpResponseMessage response = await client.GetAsync(uri, cancellationToken).ConfigureAwait(false);
```

Comment:
so, the response is no longer explicitly disposed ?

Reply:
Not as part of this call. It's up to the caller to dispose of the Response if they want it disposed quickly. If they don't dispose of it than the GC will pick it up.

I thought about just reading the response into a MemoryStream and returning that but if you inspect the internals of HttpContent it doesn't always return a MemoryStream so it seemed like a bad idea to just force a full read every time

If we don't like returning the Disposable we could deprecate this Method and force the user to pass in an Action so we control the dispose but I think that would be overkill

Reply:
> It's up to the caller to dispose of the Response

but they don't have any handle to do so

> If they don't dispose of it than the GC will pick it up

eventually, yeah

Reply:
The lifetime of the response is tied to the Stream via the returned StreamWrapper class
https://github.com/xamarin/Xamarin.Forms/pull/7431/files#diff-97e15c2e8d99d55029061029f91ea247R662

https://github.com/xamarin/Xamarin.Forms/pull/7431/files#diff-daba13799e58bccabcbccdc0583fdfa4R75